backend/utils.py

```
@@ -183,29 +183,82 @@ def extract_diy_metadata(vision_data: dict, name: str, category: str):
+    
+    return True, ""
+
+def _sanitize_username(username: str) -> str:
+    """Sanitize username to prevent SQL injection in prompts"""
+    # Remove any SQL injection characters
+    import re
+    # Only allow alphanumeric, underscore, hyphen, and dot
+    sanitized = re.sub(r'[^a-zA-Z0-9_\-.]', '', username)
+    return sanitized
+
 def chat_with_database(username: str, message_text: str):
```

Ruff (0.15.12)

[warning] Too many return statements (7 > 6)

(PLR0911)